### ISSUE TITLE
Option to Round vs Truncate MGRS/UTM

### ISSUE THREAD

**Issue by Tronald** [Tronald/CoordinateSharp#163]
UTM/MGRS coordinates are truncated as is standard to put the coordinate in the bottom left of the specified box/square depending upon precision.

This can cause issues with shifting precision loss during convert backs to Lat/Long when users must make many conversions back and forth. An option to work in rounded vs truncated would solve this problem and should be an available global setting.

[Reference Issue 162](https://github.com/Tronald/CoordinateSharp/issues/162)

**Comment by Tronald** [comment]
Enhancement should include UTM/MGRS centimeter support to include output strings (example `mgrs.ToCentimeterString()`) and parsing ability. 

This enhancement is proving to need for more accurate precision during system conversion.

**Comment by Tronald** [comment]
MGRS Truncation removed from UTM->MGRS conversion in preparation for this feature. Centimeter values will now store in `Northing` and `Easting` properties (similar to UTM). These properties were already `double`s and accepted manually entered centimeter values so the change should not have adverse effects, but could if systems excepting truncated numbers when calling values directly do not cast to `int` or floor the double value.

**Comment by Tronald** [comment]
Released with v2.8.1.1

```C#
Coordinate c = new Coordinate(40.57682, -70.75678);
c.MGRS.ToRoundedString(); // Outputs 19T CE 51308 93265
c.MGRS.ToCentimterString(); // Outputs 19T CE 51307.55707 93264.83597
```
